api/server/router/container/container_routes.go

```diff
@@ -372,6 +372,14 @@ func (s *containerRouter) postContainersCreate(ctx context.Context, w http.Respo
 	version := httputils.VersionFromContext(ctx)
 	adjustCPUShares := version.LessThan("1.19")
 
+	// set container policies associated by authz plugin(s)
+	policies, ok := ctx.Value("policies").(map[container.PolicyType]string)
+	if !ok {
+		logrus.Warnf("incorrect type for policy map %T, expected %T", ctx.Value("policies"), config.Policies)
```

Comment:
curious why warn here? if policies are not associated with the ctx, then it is ok right?